services/base-json.spec.js

```diff
@@ -35,44 +35,44 @@ class DummyJsonService extends BaseJsonService {
 
 describe('BaseJsonService', function() {
   describe('Making requests', function() {
-    let sendAndCacheRequest, serviceInstance
+    let sendAndCacheRequest
     beforeEach(function() {
       sendAndCacheRequest = sinon.stub().returns(
         Promise.resolve({
           buffer: '{"some": "json"}',
           res: { statusCode: 200 },
         })
       )
-      serviceInstance = new DummyJsonService(
-        { sendAndCacheRequest },
-        { handleInternalErrors: false }
-      )
     })
 
     it('invokes _sendAndCacheRequest', async function() {
-      await serviceInstance.invokeHandler({}, {})
+      await DummyJsonService.invoke(
+        { sendAndCacheRequest },
+        { handleInternalErrors: false }
+      )
```

Comment:
The above is an example of the changed API. `new DummyJsonService` followed by `serviceInstance.invokeHandler` are replaced by a single call to `DummyJsonService.invoke`.